doc/examples/aseprite/lib/main.dart

```diff
@@ -13,14 +13,16 @@ void main() async {
 class MyGame extends BaseGame {
   MyGame(Size screenSize) {
     size = screenSize;
-    _start();
   }
 
-  void _start() async {
+  @override
+  Future<void> onLoad() async {
+    final image = await Flame.images.load('chopper.png');
     final animation = await SpriteAnimation.fromAsepriteData(
-      'chopper.png',
+      image,
```

Comment:
this is already a static async method, so we can have both options here

Reply:
Didn't understood your suggestion.

Reply:
exactly. but lets think about that on a followup.